test/kpod_push.bats

```diff
@@ -84,3 +84,19 @@ function setup() {
     run ${KPOD_BINARY} $KPOD_OPTIONS rmi "$ALPINE"
     echo "$output"
 }
+
+@test "push with manifest type conversion" {
+    run ${KPOD_BINARY} $KPOD_OPTIONS push "${BB}" --format oci alpine dir:my-dir
+    echo "$output"
+    [ "$status" -eq 0 ]
+    run grep "application/vnd.oci.image.config.v1+json" my-dir/manifest.json
+    echo "$output"
+    [ "$status" -eq 0 ]
+    run ${KPOD_BINARY} $KPOD_OPTIONS push "${BB}" --format v2s2 alpine dir:my-dir
+    echo "$output"
+    [ "$status" -eq 0 ]
+    run grep "application/vnd.docker.distribution.manifest.v2+json" my-dir/manifest.json
+    echo "$output"
+    [ "$status" -eq 0 ]
+    rm -rf my-dir
```

Comment:
Can a test or three be added with the compression flag in play?

Reply:
I have not been able to find an uncompressed image that I can pull and compress, every image I have tried is already compressed and applying the compress flag makes no difference. If you know of an image I can pull and try please let me know.